programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -464,6 +467,14 @@ pub fn create_program_runtime_environment_v1<'a>(
         SyscallAltBn128Compression::vm,
     )?;
 
+    // Get Epoch Stake
+    register_feature_gated_function!(
+        result,
+        enable_syscall_get_epoch_stake,
+        *b"sol_syscall_get_epoch_stake",
```

Comment:
```suggestion
        *b"sol_get_epoch_stake",
```
Looks like we typically don't include `sysvall` in the name, since it's basically always redundant.